### PR TITLE
more logs for route refresh 

### PR DESCRIPTION
**Pull request by VysotskiVadim**
CP https://github.com/mapbox/mapbox-navigation-android/pull/6163

### CODE REVIEW

**Review by Guardiola31337 — APPROVED** [review]

**Comment by VysotskiVadim** [comment]
oh, I thought it would wait for CI before merging

**Comment by VysotskiVadim** [comment]
> oh, I thought it would wait for CI before merging

all required checks are green ✅

**Pull request merged**
